go/vt/vttablet/onlineddl/executor.go

```diff
@@ -3127,6 +3171,7 @@ func (e *Executor) RetryMigration(ctx context.Context, uuid string) (result *sql
 	if err != nil {
 		return nil, err
 	}
+	defer e.triggerNextCheckInterval()
```

Comment:
kick next scheduler iteration of the scheduler sooner, given we have just asked to retry a migration.